tests/deploy-minikube.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
 # shellcheck disable=SC1090
-source "$(dirname "$0")"/../scripts/resources.sh
+source "$(dirname "$0")"/../pattern-ci/scripts/resources.sh
 
 setup_dind-cluster() {
```

Comment:
We can now remove this part, and remove the one in line 46

Reply:
Also, `scripts/quickstart.sh` will need to be modified to use `manifests/istio-gateway.yaml` instead of `manifests/ingress.yaml`

Reply:
`kubectl apply -f install/kubernetes/istio.yaml`  
to  
`kubectl apply -f install/kubernetes/istio-demo.yaml`